pkg/handlers/scripts/install.sh

```diff
@@ -10,6 +10,47 @@ function fail {
 	echo "Error: $msg" 1>&2
 	exit 1
 }
+function prompts_can_prompt {
+	# Need the TTY to accept input and stdout to display
+	# Prompts when running the script through the terminal but not as a subshell
+	if [ -t 1 ] && [ -c /dev/tty ]; then
+		return 0
+	fi
+	return 1
+}
+function read_installation_path {
+	if ! prompts_can_prompt ; then
+		echo "Shell is not interactive. Using default path ${OUT_DIR}."
+		return
+	fi
+	max_attempts=3
+	while max_attempts=$((max_attempts-1)) && [ $max_attempts -ge 0 ]; do
+		printf "Enter installation path (leave blank for ${OUT_DIR}): "
+		read -r -p "" filepath </dev/tty
+		filepath="${filepath:-$OUT_DIR}"
+		if [ -d "$filepath" ] ; then
+			break
+		fi
+		printf "$filepath is not a directory...\n"
+	done
+	if [ $max_attempts -lt 0 ]; then
+		fail "3 invalid attempts. Exiting..."
```

Comment:
It's good to spell out numbers when they start a sentence.
```suggestion
		fail "Three invalid attempts. Exiting..."
```